draft-scenarios/clash/gold_rush.tex

```diff
@@ -1,7 +1,11 @@
 % !TeX spellcheck = en_US
 \addscenariosection{1}{Solo/Clash/Alliance Scenario}{Gold Rush}{\images/gold-mine.png}
 
-\begin{multicols}{2}
+\newenvironment{Figure}
+{\par\medskip\noindent\minipage{\linewidth}}
+{\endminipage\par\medskip}
+
```

Comment:
What's that for?

Reply:
Tried different ideas to make putting Map Layouts easier. I will remove it.